dev/docker/trino/conf/node.properties

```diff
@@ -3,5 +3,6 @@
 # This software is licensed under the Apache License version 2.
 #
 node.environment = docker
+node.id = ffffffff-ffff-ffff-ffff-ffffffffffff
```

Comment:
I recall that this line was added in the first edition. Please see https://github.com/datastrato/gravitino/issues/434
<img width="1027" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/65bd5795-7dfa-4167-9893-d968899c16e0">